unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/Channels/ChatChannelsControllerMock.cs

```diff
@@ -293,5 +327,64 @@ private async UniTask MuteFakeChannel(string channelId)
             };
             controller.UpdateChannelInfo(JsonUtility.ToJson(msg));
         }
+
+        private async UniTask SimulateDelayedResponseFor_ChatInitialization()
+        {
+            await UniTask.Delay(Random.Range(50, 1000));
+
+            var payload = new InitializeChatPayload
+            {
+                totalUnseenPrivateMessages = Random.Range(0, 5)
+            };
+            controller.InitializeChat(JsonUtility.ToJson(payload));
+        }
+
+        private async UniTask SimulateDelayedResponseFor_ChannelsInitialization()
+        {
+            await UniTask.Delay(Random.Range(50, 1000));
+
+            var payload = new InitializeChannelsPayload
+            {
+                unseenTotalMessages = Random.Range(0, 5)
+            };
+            controller.InitializeChannels(JsonUtility.ToJson(payload));
+        }
+
+        private async UniTask SimulateDelayedResponseFor_MarkChannelAsSeen(string channelId)
+        {
+            await UniTask.Delay(Random.Range(50, 1000));
+
+            var totalPayload = new UpdateTotalUnseenMessagesPayload
+            {
+                total = Random.Range(0, 10)
+            };
+            controller.UpdateTotalUnseenMessages(JsonUtility.ToJson(totalPayload));
```

Comment:
im thinking that we have a redundancy over the `InitializeChat `/`InitializeChannels ` messages, since we are sending the same data here.. should we simplify the total unseen messages it to only one message? `UpdateTotalUseenMessages`

Reply:
Yeah, maybe we could join both initializations in one. We can discuss that next monday together 👍 

Anyway these initializations are not currently getting the same information, the `InitializeChat` is giving us the total amount of unseen PRIVATE messages, and the `InitializeChannels` is giving us the total unseen CHANNEL messages.

But yeah, I think we could have both in a same place 👍